OracleAnalytics/patches/6.4-patch/Dockerfile

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2022 Oracle and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This Dockerfile extends the Oracle Analytics Server 6.4, and applies necessary
+# patches required.
+
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) p34080315_122140_Generic.zip
+# (2) p33958532_122140_Generic.zip
+# (3) p34044738_122140_Generic.zip
+# (4) p32784652_122140_Generic.zip
+# (5) p30613424_122140_Generic.zip
+# (6) p31403376_122140_Generic.zip
+# (7) p33618954_122140_Generic.zip
+# (8) p33546536_12214211129_Generic.zip
+# (9) p32575741_122140_Linux-x86-64.zip
+# (10) p33950717_122140_Generic.zip
+# (11) p34065178_122140_Generic.zip
+
+#     Download the patches from https://support.oracle.com
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build --force-rm=true --no-cache=true -t oracle/fmw-infrastructure:12.2.1.4-patched-for-oas64 .
+#
+
+# Pull base image
+# ---------------
+FROM oracle/analyticsserver:6.4
+
+# Maintainer
+# ----------
+MAINTAINER Padam Bengani <padam.bengani@oracle.com>
```

Comment:
The `MAINTAINER` tag is deprecated. Please use `LABEL` directives as documented in the [contribution guide](https://github.com/oracle/docker-images/blob/main/CONTRIBUTING.md).